rewrite-maven/src/main/java/org/openrewrite/maven/tree/ResolvedPom.java

```diff
@@ -325,7 +345,7 @@ void resolveParentsRecursively(Pom requested) throws MavenDownloadingException {
             if (initialRepositories != null) {
                 mergeRepositories(initialRepositories);
             }
-            resolveParentPropertiesAndRepositoriesRecursively(pomAncestry);
+            resolveParentPropertiesAndRepositoriesRecursively(new ArrayList<>(pomAncestry));
```

Comment:
this avoid the need to clear the list and populate it after every call to a `resolveParent*Recursively`, which was missing for resolveParentPluginManagementRecursively causing not proper resolution.